Sources/SwiftDocC/SwiftDocC.docc/Resources/ThemeSettings.spec.json

```
@@ -856,6 +856,9 @@
                                     },
                                     "onThisPageNavigator": {
                                         "$ref": "#/components/schemas/FeatureFlagDisable"
+                                    },
+                                    "i18n": {
+                                        "$ref": "#/components/schemas/FeatureFlagDisable"
```

i18 is defined by "enable"
```suggestion
                                        "$ref": "#/components/schemas/FeatureFlagEnable"
```

---

See https://github.com/apple/swift-docc-render/blob/95129994a3e5e3ccb48ad8cd2ca4fd38f91e9afe/src/App.vue#L98

---

Thanks for looking into it @Kyle-Ye ! I fixed it https://github.com/apple/swift-docc/pull/475/commits/d2fba163f0c0534c022053cd7c950b377c29afc2